test/versioned-external/external-repos.js

```diff
@@ -20,7 +20,7 @@ const repos = [
   {
     name: 'koa',
     repository: 'https://github.com/newrelic/node-newrelic-koa.git',
-    branch: 'main'
+    branch: 'update-get-shim'
```

Comment:
Is there a PR for this I should also take a look at?

Reply:
no that was merged [here](https://github.com/newrelic/node-newrelic-koa/pull/149). ill update, nice catch!